mailpoet/assets/css/src/mailpoet-admin.scss

```diff
@@ -3,3 +3,19 @@
 
 @import 'components-admin/menu';
 @import 'components-admin/3rd-party-plugins/members';
+
+// Generic
+// 3rd-party styles, can be overwritten.
+
+@import '../../../node_modules/@wordpress/interface/build-style/style';
+@import '../../../node_modules/@wordpress/editor/build-style/style';
+@import '../../../node_modules/@wordpress/components/build-style/style';
+@import '../../../node_modules/@wordpress/edit-post/build-style/style';
+@import '../../../node_modules/@wordpress/edit-post/build-style/classic';
+@import '../../../node_modules/@wordpress/block-editor/build-style/style';
+@import '../../../node_modules/@wordpress/block-library/build-style/style';
+@import '../../../node_modules/@wordpress/block-library/build-style/theme';
+@import '../../../node_modules/@wordpress/block-library/build-style/editor';
+@import '../../../node_modules/@wordpress/format-library/build-style/style';
+@import '../../../node_modules/codemirror/lib/codemirror';
+@import '../../../node_modules/codemirror/theme/neo';
```

Comment:
Here we are moving all styles coming from WordPress packages to the main MailPoet admin CSS file. We use only a single component in the subscriber's form. Could we move only the @wordpress/components styles instead of moving all of them?